bug fix change VAT depends on country selection

DanielMaass opened the pull request:
-[CORE-802]

[CORE-802]: https://evannetwork.atlassian.net/browse/CORE-802

Commit:
bug fix change VAT depends on country selection

-[CORE-802]

karladler commented on dapps/profile.vue/src/components/wallet/buy/BuyEve.ts:
```diff
@@ -107,6 +108,12 @@ export default class BuyEveComponent extends mixins(EvanComponent) {
+  @Watch('contactForm.country.value')
+  onCountryChange(country) {
+    // change tax value depends on country select
+    this.taxValue = country === 'DE' && !this.contactForm.vat.value ? 19 : 0;
```
This looks like it's possible to buy from Germany without paying VAT. IMO this shouldn't be the behaviour.

karladler replied:
see also: https://evannetwork.atlassian.net/browse/CORE-395

S3bb1 commented on dapps/profile.vue/src/components/wallet/buy/BuyEve.ts:
```diff
@@ -107,6 +108,12 @@ export default class BuyEveComponent extends mixins(EvanComponent) {
+  @Watch('contactForm.country.value')
+  onCountryChange(country) {
+    // change tax value depends on country select
+    this.taxValue = country === 'DE' && !this.contactForm.vat.value ? 19 : 0;
```
```suggestion
    this.taxValue = country === 'DE' ? 19 : 0;
```

Commit:
Update dapps/profile.vue/src/components/wallet/buy/BuyEve.ts

karladler committed:
fix validate function

- [CORE-802] use german VAT amount every time a foreign VAT-ID is invalid or doesn't match the companies HQ country

datrinh commented:
Could we at least put the hardcoded numbers into constant variables please? :)

karladler committed:
Merge branch 'refs/heads/develop' into feature/CORE-802-bug-fix-ui-test-payment

# Conflicts:
#	dapps/assets/src/components/digitaltwins/DigitalTwins.ts

karladler committed:
Add German VAT as constant, fix bug for user profile

- [CORE-802, CORE-804]
- PR requested changes
- Fix for bug, when buying EVE with unspecified account
